unlock-app/src/components/lock/Lock.js

```diff
@@ -39,18 +40,21 @@ export const Lock = ({
     const soldOut =
       lock.outstandingKeys >= lock.maxNumberOfKeys &&
       lock.maxNumberOfKeys !== UNLIMITED_KEYS_COUNT
+    const tooExpensive =
+      account && parseFloat(account.balance) <= parseFloat(lock.keyPrice)
```

Comment:
Actually @cellog This is already safe?

Reply:
Indeed! Take the wasted time and emotional pain out of my next paycheck, apologies :)